### PR TITLE
chore: Documents new configuration options for the .NET agent

### PR DESCRIPTION
**Pull request by tippmar-nr**
Updates the configuration documentation for the .NET agent to include 3 new configuration options introduced in v10.21.0.

### CODE REVIEW

**Comment by akristen** [comment]
Hi @tippmar-nr ! Thanks for starting this draft PR. When you're ready for a review and a merge, send us a message in our docs Slack channel.

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 1]
```diff
@@ -1258,6 +1258,124 @@ The `log` element supports the following attributes:
...
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    Defines the agent log file rolling strategy. Possible values are: 
+    * `size` (default): The log file rolls over when it reaches the size specified in `maxLogFileSizeMB`.
+    * `day`: The log file rolls over at midnight.
+
+    Alternatively, set the `NEW_RELIC_LOG_ROLLING_STRATEGY` environment variable in the application's environment.
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:1294]:
```suggestion
    Alternatively, set the `NEW_RELIC_LOG_ROLLING_STRATEGY` environment variable in the application's environment:
```

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 2]
```diff
@@ -1258,6 +1258,124 @@ The `log` element supports the following attributes:
...
+          <td>
+            4
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    Defines the number of agent log files to retain. When the maximum number of log files is reached, the oldest log file is deleted. A value of `0` specifies that there is no maximum number of log files.
+
+    Alternatively, set the `NEW_RELIC_LOG_MAX_FILES` environment variable in the application's environment.
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:1332]:
```suggestion
    Alternatively, set the `NEW_RELIC_LOG_MAX_FILES` environment variable in the application's environment:
```

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 3]
```diff
@@ -1258,6 +1258,124 @@ The `log` element supports the following attributes:
...
+          <td>
+            50
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    Defines the maximum size of the agent log file in megabytes before it rolls over. Valid only when `logRollingStrategy` is set to `size`. A value of `0` specifies that there is no maximum log file size.
+
+    Alternatively, set the `NEW_RELIC_LOG_MAX_FILE_SIZE_MB` environment variable in the application's environment.
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:1371]:
```suggestion
    Alternatively, set the `NEW_RELIC_LOG_MAX_FILE_SIZE_MB` environment variable in the application's environment:
```

**Review by ghost — APPROVED** [review]

**Pull request merged**